docs/changelog.md

```diff
@@ -49,11 +104,20 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
  - `ScEvent` class was renamed to `ScEventSubscription`
  - `ScEventParams` class was renamed to `ScEventSubscriptionParams`
 
+### Fixed
+ - Checking of all syntactic and semantic subtypes for types in ScType `merge` method.
+ - Now sc-link is sc-node
```

Comment:
```suggestion
 - sc-link is sc-node
```

Reply:
No

Reply:
all entries in changelog mean that it is already in the project, why don't you add 'now' to all of them then?

Reply:
`sc-link is sc-node` is not understandable mention for users